kingfisher_scrapy/spiders/kenya_makueni.py

```diff
@@ -18,14 +17,19 @@ class KenyaMakueni(IndexSpider):
     # SimpleSpider
     data_type = 'release_package'
 
-    # IndexSpider
-    count_pointer = ''
-    limit = 1000  # > 1000 causes "must be between 1 and 1000"
-    use_page = True
-    start_page = 0
-    param_page = 'pageNumber'
-    base_url = f'https://opencontracting.makueni.go.ke/api/ocds/package/all?pageSize={limit}'
-
     def start_requests(self):
-        url = f'https://opencontracting.makueni.go.ke/api/ocds/release/count?pageSize=1000'
-        yield scrapy.Request(url, meta={'file_name': 'count.json'}, callback=self.parse_list)
+        yield from self.request_page()
+
+    @handle_http_error
+    def parse(self, response):
+        yield from super().parse(response)
+
+        if len(response.json()) > 0:
```

Comment:
`[]` is already falsy.

```suggestion
        if response.json():
```

Reply:
Yes, you're right